pyperformance/data-files/benchmarks/bm_alloc/pyproject.toml

```
@@ -0,0 +1,9 @@
+[project]
+name = "pyperformance_bm_alloc"
+requires-python = ">=3.8"
+dependencies = ["pyperf"]
+urls = {repository = "https://github.com/python/pyperformance"}
+dynamic = ["version"]
+
+[tool.pyperformance]
+name = "alloc"
```

Please tag it as a micro benchmark.

```suggestion
[tool.pyperformance]
name = "alloc"
tags = "micro"
```

---

I have added the micro benchmark tag